build.sbt

```diff
@@ -104,7 +105,7 @@ val flagsFor11 = Seq(
 val flagsFor12 = Seq(
   "-Xlint:_",
   "-Ywarn-infer-any",
-  "-opt:l:project"
+  "-opt:l:inline"
```

Comment:
This should be `"-opt-inline-from:<sources>"`. See https://github.com/mdedetrich/scalajson/issues/43